### ISSUE TITLE
DOS: Tiny memory model (-mt switch) gives error

### ISSUE THREAD

**Issue by darkstar** [open-watcom/open-watcom-v2#275]
While trying to compile FreeDOS kernel with latest Watcom SVN/GIT source code, I came across this bug.

The "-mt" switch (to select the "tiny" memory model) does not work (it gives an error about an invalid option). the "-ms" option works though, for example

Log (this is the command that the FreeDOS build system uses. I edited it only for clarity):
```
C:\SRC\FREEDOS\KERNEL\UTILS>wcl -zq -zp1 -os -s -we -e3 -wx -bt=DOS -mt -I..\hdr patchobj.c
Error! E1073: Invalid option '-mt'
Error: Compiler returned a bad status compiling "PATCHOBJ.C"
C:\SRC\FREEDOS\KERNEL\UTILS>wcl -zq -zp1 -os -s -we -e3 -wx -bt=DOS -ms -I..\hdr patchobj.c
C:\SRC\FREEDOS\KERNEL\UTILS>
```

And for the version
```
C:\SRC\FREEDOS\KERNEL\UTILS> wcl -version
Open Watcom C/C++ x86 16-bit Compile and Link Utility
Version 2.0 beta Aug 22 2016 15:22:13 (16-bit)
Copyright (c) ...
...
```
The git revision is
```
darkstar@kubuntu:/usr/src/open-watcom$ git branch -v
* master 67daa55 correct use of macros for remapping between dig_fhandle type and Windows HANDLE, POSIX handle and Debugger file handle use DIGCli... function for open/close debug info files
```

**Comment by darkstar** [comment]
Additional Info: It looks like it's wcc that chokes on the "-mt", even though the wcl binary (according to the source code) should correctly parse the "-mt" switch (it switches to "-ms" and sets a flag for the tiny model: https://github.com/open-watcom/open-watcom-v2/blob/master/bld/wcl/c/wcl.c#L776 )

Here's a screenshot of the command without "-zq" (a bit more verbose):
http://imgur.com/a/rUKd5

**Comment by jmalak** [comment]
Thanks for bug report.
it is now fixed by f76af83c2e9cd10b34e6964ab3c71c60ba4277f0
I fixed bug if no name is specified on command line by c39f28f659f5c3bd643893c3bdcc8c07361c025f

**Comment by darkstar** [comment]
Thanks, that fixed it. closing this issue
